CompuCell3D/core/CompuCell3D/steppables/PDESolvers/ReactionDiffusionSolverFVM.h

```diff
@@ -1159,6 +1162,16 @@ namespace CompuCell3D {
 		 * @return false 
 		 */
 		bool isValid(Point3D pt) { try { pt2ind(pt); return true; } catch (CC3DException) { return false; } }
+
```

Comment:
maybe we should try adding c++ docstrings to other places in the code? I like it